v3/newrelic/tracing.go

```diff
@@ -206,12 +210,16 @@ func (b boolJSONWriter) WriteJSON(buf *bytes.Buffer) {
 // value is a jsonWriter to allow for segment query parameters.
 type spanAttributeMap map[string]jsonWriter
 
+// addString writes the span attribute with any value over passed in limit
```

Comment:
This comment should be updated since we are no longer directly passing in the `limit` value